templates/CRM/Admin/Form/Job.tpl

```diff
@@ -92,7 +92,8 @@ CRM.$(function($) {
     <tr class="crm-job-form-block-scheduled-run-date">
         <td class="label">{$form.scheduled_run_date.label}</td>
         <td>{$form.scheduled_run_date.html}<br />
-            <div dlass="description">{ts}Do not run this job before this date / time. The run frequency selected above will apply thereafter.{/ts}<br />
+            <div class="description">{ts}Do not run this job before this date / time. The run frequency selected
+              above will apply thereafter.{/ts}<br />
```

Comment:
The dlass => class change is good but `ts` doesn't work very well when lines are split. Can you put it back on one line?

Reply:
thanks for pointing it out! fixed!